third_party/parser/include/ruby_parser/lexer.hh

```diff
@@ -26,6 +26,7 @@ enum class ruby_version {
     RUBY_23,
     RUBY_24,
     RUBY_25,
+    RUBY_26,
```

Comment:
This is useless as there are no conditionals based on 2.6 version in the lexer right now but I will add `RUBY_27` after and it seemed odd to jump a version.